docs/overview/getting-started/avax.md

```diff
@@ -0,0 +1,26 @@
+# AVAX
+
+AVAX is the native utility token of Avalanche. It’s a hard-capped, scarce asset that is used to 
+pay for fees, secure the platform through staking, and provide a basic unit of account between the
+multiple Subnets created on Avalanche. `1 nAVAX` is equal to `0.000000001 AVAX`.
+
+## Utility
+
+AVAX is a non-renewable resource (analogous to oil) that's used to power the Avalanche network.
+AVAX is used for day-to-day operations in Avalanche such as staking and issuing transactions.
+
+AVAX represents the weight that each node has in network decisions. No single actor owns 
+the Avalanche Network, so each validator in the network is given a proportional weight in the 
+network's decisions corresponding to the proportion of total stake that they own.
+
+Any entity trying to execute a workload on Avalanche pays a corresponding fee (commonly known as 
+"gas") to run it on the network. The gas used to execute a workload on Avalanche is removed from
+existence (commonly known as "burning"). 
+
+A small portion of burnt gas fees are used to reward validators at the end of their staking 
+periods. Validators are rewarded for virtuous (e.g not attempting lie, steal, or otherwise
```

Comment:
Can we make it clear that the burnt fees do not go towards validator rewards since these are two separate things? This reads as if X% of burnt fees are used to reward validators, which is not the case on the P-Chain.

Reply:
agree, we should probably also add that X amount of AVAX was minted at genesis and then add the equation for distribution over time.